tests/unittests/Foundation/ReferenceFoundation/TestNSURLResponse.mm

```diff
@@ -71,21 +71,35 @@
     ASSERT_OBJCEQ(nil, res2.textEncodingName);
 }
 
-TEST(NSURLResponse, SuggestedFilename) {
+// Disabled pending GH#832; we do not respect the incoming MIME type when generating filenames. OS X does.
+OSX_DISABLED_TEST(NSURLResponse, SuggestedFilename) {
     NSURL* url = [NSURL URLWithString:@"a/test/name.extension"];
-    NSURLResponse* res = [[[NSURLResponse alloc] initWithURL:url MIMEType:@"txt" expectedContentLength:0 textEncodingName:nil] autorelease];
+    NSURLResponse* res = [[[NSURLResponse alloc] initWithURL:url MIMEType:@"text/plain" expectedContentLength:0 textEncodingName:nil] autorelease];
     ASSERT_OBJCEQ(@"name.extension", res.suggestedFilename);
 }
 
-TEST(NSURLResponse, SuggestedFilename_2) {
```

Comment:
(I know they're not very good names, but)

---
In reply to: [73769571](https://github.com/Microsoft/WinObjC/pull/835#discussion_r73769571) [](ancestors = 73769571)